### PR TITLE
Added recipe for sciluigi

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/sciluigi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - sciluigi
+
+about:
+  home: https://github.com/samuell/sciluigi
+  # license_file: No MANIFEST.in - see https://github.com/pharmbio/sciluigi/pull/35
+  # Manually including a copy.
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (ericdill) [inline recipes/sciluigi/meta.yaml]: I've also seen this done as `{{ RECIPE_DIR }}/LICENSE`. Is one way preferred over the other?

File: recipes/sciluigi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - luigi >=1.3,<=2.1
+
+test:
+  imports:
+    - sciluigi
+
+about:
+  home: https://github.com/samuell/sciluigi
+  # license_file: No MANIFEST.in - see https://github.com/pharmbio/sciluigi/pull/35
```
Comment (ericdill) [inline recipes/sciluigi/meta.yaml]: :+1:

File: recipes/sciluigi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  # Manually including a copy.
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  license: MIT
+  license_family: MIT
+  summary: 'Helper library for writing dynamic, flexible workflows in luigi'
+  dev_url: https://github.com/samuell/sciluigi
```
Comment (ericdill) [inline recipes/sciluigi/meta.yaml]: Please use the same url (even though samuell apparently redirects to pharmbio)

File: recipes/sciluigi/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/sciluigi/meta.yaml]: I hadn't seen the other way. If that works, I prefer it because it would look cleaner.

**Comment by ericdill** [comment]
Thanks @pmlandwehr . LGTM. Merge away once Appveyor is ✅

**Pull request merged**